integration/integration-setup.ts

```diff
@@ -44,7 +44,9 @@ export async function WithCache(
 
 export const CacheClientProps: SimpleCacheClientProps = {
   configuration: Configurations.Laptop.latest(),
-  credentialProvider: new EnvMomentoTokenProvider('TEST_AUTH_TOKEN'),
+  credentialProvider: new EnvMomentoTokenProvider({
+    environmentVariableName: 'TEST_AUTH_TOKEN',
+  }),
```

Comment:
I'm having deja vu. Didn't we just do this?
https://github.com/momentohq/client-sdk-javascript/pull/204/files

Reply:
sorry it's because the parent branch got squash-merged.  This is rebased now.

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/205/commits/15e2d8ea91e9977d19ba25e87b865123dd5c8b28